packages/variable-editor/src/components/variables/master/AddVariableDialog.css

```diff
@@ -1,3 +1,11 @@
 .add-variable-dialog-trigger-button {
   margin-left: auto;
 }
+
+.ui-combobox-menu {
+  z-index: 50;
+}
```

Comment:
I think this should not be needed any longer. 
Try to update to the latest ui-components.

Reply:
You are right, but I notice that after updating the toolbar also stays while the dialog is open :sweat_smile: 
![image](https://github.com/axonivy/config-editor-client/assets/141232142/421e7d51-6ccd-44f0-be93-38b9dfd2ac6c)

Reply:
:(
sadly this has also a zindex... Need to remove this. Hopefully which no side effect :D